jiratools/__init__.py

```diff
@@ -177,7 +177,7 @@ def _list_from_config(key_name: str) -> List[str]:
     _load_config()
     if not CONFIG:
         raise ConfigNotFoundException
-    return list(filter(None, [x.strip() for x in CONFIG.get(key_name, "").split(",")]))
+    return list(filter(None, (x.strip() for x in CONFIG.get(key_name, "").split(","))))
```

Comment:
Thanks `flake8-tuple`, I guess? 😆

Reply:
Yeah, I mean, _technically_ it would be more efficient at a large scale?

Reply:
Nah, that's a generator expression https://docs.python.org/3/reference/expressions.html#generator-expressions
(the parenthesis around generator expressions are required except in a few instances)